### DISCUSSION TITLE
How to analyze charts?

### DISCUSSION THREAD

**Discussion by MaksimSerg** [coqui-ai/TTS#1660] [General Q&A]
Hi! How to analyze charts?
![1](https://user-images.githubusercontent.com/50648550/174004860-2c67353f-ea6e-4a24-b549-dded5891df6b.png)
![2](https://user-images.githubusercontent.com/50648550/174004867-d528df6a-428b-4ee1-9520-971279f00805.png)
![3](https://user-images.githubusercontent.com/50648550/174004873-52ce9897-6f99-489c-b4f2-0ecf0a33cfd6.png)
![4](https://user-images.githubusercontent.com/50648550/174004878-d9bed482-8b02-4dfc-8b54-c68dacd0e7fb.png)

**Comment by WeberJulian** [comment]
This is heavily model specific, but in general -> Eval loss goes down = good news :) When it stops going down, you should stop the training. For GAN based architecture it's more complicated...
